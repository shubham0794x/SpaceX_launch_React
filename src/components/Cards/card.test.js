import React from "react";
import { render as testRender } from "react-dom";
import Card from "../card";

it("should match the snapshot", () => {
  const fakeLaunchData = {
    flight_number: "1",
    mission_name: "test name",
    mission_id: ["123"],
    launch_success: true,
    launch_date_local: "2020-08-24T15:59:15.582Z",
    rocket: {
      first_stage: {
        cores: [{ land_success: false }],
      },
    },
    links: {
      mission_patch_small: "https://images2.imgbox.com/4f/e3/I0lkuJ2e_o.png",
    },
  };

  const cardTree = <Card event={fakeLaunchData} />;
  expect(cardTree).toMatchSnapshot();
});
