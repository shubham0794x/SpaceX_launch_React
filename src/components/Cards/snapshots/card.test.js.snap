// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match the snapshot 1`] = `
<Card
  event={
    Object {
      "flight_number": "1",
      "launch_date_local": "2020-08-24T15:59:15.582Z",
      "launch_success": true,
      "links": Object {
        "mission_patch_small": "https://images2.imgbox.com/4f/e3/I0lkuJ2e_o.png",
      },
      "mission_id": Array [
        "123",
      ],
      "mission_name": "test name",
      "rocket": Object {
        "first_stage": Object {
          "cores": Array [
            Object {
              "land_success": false,
            },
          ],
        },
      },
    }
  }
/>
`;
